lib/carraway/config.rb

```diff
@@ -6,6 +6,8 @@ class << self
       def load(file)
         @config = YAML.load_file(file)
         Category.load(@config['categories'])
+
+        @root = ::File.expand_path('..', file)
```

Comment:
Custom view option determines configuration file path as a view root path?
I think this behavior is confusing and easy to cause unexpected behavior.

Reply:
I think so, so I use carraway working directory as a view root path.